bin/ch/ch.cpp

```diff
@@ -671,6 +671,10 @@ HRESULT ExecuteTest(const char* fileName)
         }
     }
 Error:
+    if( fileContents != nullptr)
+    {
+        free((char*)fileContents);
```

Comment:
I meant the ExternalArrayBuffer for the bytecode buffer created to call JsParseSerializedScript, not ExternalArrayBuffer for the script source created in the loadCallback. When the bytecode ExternalArrayBuffer is being finalized, you "no longer need the ability to construct the source".